app/assets/stylesheets/application.scss

```diff
@@ -4,7 +4,8 @@ $govuk-page-width: 1140px;
 @import "user_research_recruitment_banner";
 
 // Components from this application
-@import "components/*";
+@import "components/_contact-details";
+@import "components/_table";
```

Comment:
The [Other Issues > Globs section](https://docs.publishing.service.gov.uk/manual/migrate-to-dart-sass-from-libsass.html#globs) of the instructions suggested that the wildcard would not work, although I have to admit, I didn't try it. I'll try it now and see whether it works - I can see how it might be nicer to use a wildcard so we don't have to remember to add each component individually.

Reply:
I just tried using the wildcard again locally and I saw the following error when trying to view the app in a browser:

```
Error: Can't find stylesheet to import.
  ╷
7 │ @import "components/*";
  │         ^^^^^^^^^^^^^^
  ╵
  app/assets/stylesheets/application.scss 7:9  root stylesheet
```

So I think I'll leave the explicit individual imports in place.

Reply:
Ah, interesting. I'm afraid I managed to miss the end of the guide so hadn't noticed the section on Globs. Sorry!